tests/contrib/psycopg/test_psycopg.py

```diff
@@ -230,3 +230,13 @@ def test_backwards_compatibilty_v3():
     conn.cursor().execute("select 'blah'")
 
 
+@skipIf(PSYCOPG_VERSION < (2, 7), 'quote_ident not available in psycopg2<2.7')
+def test_manual_wrap_extension_quote_ident_standalone():
+    from ddtrace import patch_all
+    patch_all()
+    from psycopg2.extensions import quote_ident
```

Comment:
Don't we have the issue if the order is reversed? or it's the other way around?

Reply:
reordering seems to not patch `quote_ident` I'll have to dig into how python treats these object references, I would have assumed that it'd be safe to patch after importing because it's the same reference...